microsoft-edge/extensions-chromium/publish/update-extension.md

```diff
@@ -12,7 +12,7 @@ ms.date: 02/17/2021
 You can update a submitted extension or remove a published extension listing from the Microsoft Edge Add-ons website at any time.
 
 > [!NOTE]
-> The certification process for an update to an extension can take from a few hours to a few days.
+> The certification process for an update to an extension can take up to 7 business days.
```

Comment:
I can't enter a Suggestion-type Comment on the non-changed Note line, so I will push a commit.

I'm trying converting from Note to regular text, to increase its visibility per
https://docs.microsoft.com/en-us/contribute/markdown-reference#alerts-note-tip-important-caution-warning -
"Avoid Notes, Tips, and Important boxes. Readers tend to skip over them. It's better to put that info directly into the article text."